ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -245,25 +245,46 @@ function getValues(pendingApproval, t, actions, history, data) {
         ],
       },
 
-      {
-        element: 'Typography',
-        key: 'title',
-        children: originIsMetaMask
-          ? t('wantToAddThisNetwork')
-          : t('addEthereumChainConfirmationTitle'),
-        props: {
-          variant: TypographyVariant.H3,
-          align: 'center',
-          fontWeight: 'bold',
-          boxProps: {
-            margin: [0, 0, 4],
+      process.env.CHAIN_PERMISSIONS
+        ? {
+            element: 'Typography',
+            key: 'title',
+            children: originIsMetaMask
+              ? t('wantToAddThisNetwork')
+              : t('addNetworkConfirmationTitle', [
```

Comment:
Could we just use a ternary on this `children` property since the rest of the element and props seem identical?

Reply:
Done https://github.com/MetaMask/metamask-extension/pull/25777/commits/e87feb11f778de6e03ad95819414cbb2233c4b80